cmd/wego/add/cmd.go

```diff
@@ -42,7 +44,19 @@ func init() {
 
 func runCmd(cmd *cobra.Command, args []string) {
 	params.Namespace, _ = cmd.Parent().Flags().GetString("namespace")
-	if err := cmdimpl.Add(args, params); err != nil {
+
+	authMethod, err := ssh.NewPublicKeysFromFile("git", params.PrivateKey, params.PrivateKeyPass)
```

Comment:
does this means that it is strictly necessary to pass in these two flags? If passing them empty means that it will take the default ssh key then ignore my comment. Otherwise I would add as the default option to look for the default ssh key in `~/.ssh/`

Reply:
no change in current behavior, currently `params.PrivateKey` defaults to the user's private key.